compositor_render/src/transformations/web_renderer/shared_memory.rs

```diff
@@ -42,22 +36,11 @@ impl SharedMemory {
         self.path.display().to_string()
     }
 
-    pub fn ensure_size(
-        &mut self,
-        size: usize,
-        frame: &cef::Frame,
-    ) -> Result<(), SharedMemoryError> {
-        let shmem_len = self.inner.as_ref().map(|shmem| shmem.len()).unwrap();
-        if shmem_len == size {
-            return Ok(());
-        }
-
-        // TODO: This should be synchronised
-        let mut process_message = cef::ProcessMessage::new(UNEMBED_SOURCE_FRAMES_MESSAGE);
-        process_message.write_string(0, self.path.display().to_string());
-        frame.send_process_message(cef::ProcessId::Renderer, process_message)?;
-        // -----
+    pub fn len(&self) -> usize {
+        self.inner.as_ref().map(|shmem| shmem.len()).unwrap()
+    }
 
+    pub fn resize(&mut self, size: usize) -> Result<(), SharedMemoryError> {
         // Releasing the current `Shmem` instance to ensure it does not erase the shared memory descriptor from the file system
         // This is critical to ensure when a new `Shmem` is created at the same location, it doesn't conflict with the old descriptor
         drop(self.inner.take());
```

Comment:
```suggestion
        self.inner.take();
```
wouldn't this be the same?

Reply:
This whole line is not needed unless I forgot about something.